static/src/javascripts/projects/commercial/modules/mobile-sticky.js

```diff
@@ -0,0 +1,45 @@
+// @flow strict
+import config from 'lib/config';
+import fastdom from 'lib/fastdom-promise';
+import { getCookie } from 'lib/cookies';
+import { isBreakpoint } from 'lib/detect';
+import { addSlot } from 'commercial/modules/dfp/add-slot';
+import { createSlots } from 'commercial/modules/dfp/create-slots';
+import { commercialUsMobileSticky } from 'common/modules/experiments/tests/commercial-us-mobile-sticky';
+import { isInVariantSynchronous } from 'common/modules/experiments/ab';
+
+const createAdWrapper = (): HTMLDivElement => {
+    const wrapper: HTMLDivElement = document.createElement('div');
+    wrapper.className = 'mobilesticky-container';
+    const adSlot = createSlots('mobile-sticky', {})[0];
+    wrapper.appendChild(adSlot);
+    return wrapper;
+};
+
+const isInNA = (): boolean =>
+    (getCookie('GU_geo_continent') || 'OTHER').toUpperCase() === 'NA';
+
+export const init = (): Promise<void> => {
+    if (
+        isInNA() &&
```

Comment:
Hu? Isn't that for the US only? Ha I think I get it. NA = North America, not 'Not Applicable'. Please comment so confusion is lifted.

Reply:
Yes, NA stands for North America. Will leave a comment. 👍